### ISSUE TITLE
Update for stylelint-scss@2.0.0

### ISSUE THREAD

**Issue by sinedied** [kristerkari/stylelint-config-recommended-scss#3]
Since the latest `stylelint-scss@2.0.0` update I have this warning:
```
warning "stylelint-config-recommended-scss@1.0.0" has incorrect peer dependency "stylelint-scss@^1.5.1".
```

Could you consider updating your dependencies and rules to match it?
Thanks

**Comment by kristerkari** [comment]
Thanks! True, I'll update the dependency and release new version tomorrow.
